lib/ansible/modules/system/java_keystore.py

```diff
@@ -136,7 +136,9 @@ def read_certificate_fingerprint(module, openssl_bin, certificate_path):
 
 
 def read_stored_certificate_fingerprint(module, keytool_bin, alias, keystore_path, keystore_password):
-    stored_certificate_fingerprint_cmd = "%s -list -alias '%s' -keystore '%s' -storepass '%s'" % (keytool_bin, alias, keystore_path, keystore_password)
+    grep_bin = module.get_bin_path('grep', True)
+    stored_certificate_fingerprint_cmd = "%s -list -alias '%s' -keystore '%s' -storepass '%s' -v | %s SHA1" % (
```

Comment:
I think it would be more portable to update the regular expression below rather than using `grep`

Reply:
I take this way in order to preserve the unit test. I'll take a look changing the regexp.